source/ed/tests/ed2nav_test.cpp

```diff
@@ -72,11 +72,9 @@ BOOST_AUTO_TEST_CASE(writing_test_ed2nav_should_not_throw) {
     // Checks if the already existing binary is renamed and the new one is created
     BOOST_CHECK(ed::write_data_to_file(test, data));
     BOOST_CHECK(boost::filesystem::exists("ed2nav_test.nav.lz4"));
-    BOOST_CHECK(boost::filesystem::exists("ed2nav_test.nav.lz4.bak"));
```

Comment:
Why not BOOST_CHECK(!boost::filesystem::exists("ed2nav_test.nav.lz4.bak")); instead ?